app/views/admin/tags/_form.html.erb

```diff
@@ -32,9 +28,9 @@
     <%= f.label :pretty_name, class: "crayons-field__label" %>
     <%= f.text_field :pretty_name, value: tag.pretty_name, class: "crayons-textfield" %>
   </div>
-  <div class="crayons-field mt-3">
+  <div class="crayons-field crayons-field--checkbox mt-3">
```

Comment:
was this a cleanup unrelated to the category UI change?

Reply:
Yeah it was 😅 Before it looked pretty bad so I figured I'd just fix it now.

![image](https://user-images.githubusercontent.com/17884966/136264094-64a3839d-bee9-46c7-89ea-62574a3a90bb.png)

Reply:
Now it's much better:
![image](https://user-images.githubusercontent.com/17884966/136264218-286aa36d-75a1-45ec-a3d6-c4748b56f4f8.png)